keras/src/ops/core_test.py

```diff
@@ -607,6 +608,7 @@ def test_stop_gradient_return(self):
         y = ops.stop_gradient(x)
         self.assertAllClose(x, y)
 
+    @pytest.mark.openvino_backend
```

Comment:
It seems impractical to mark individual tests (at the method level as whitelisted for the openvino backend).

In general you should have fewer than ~30 backend-specific annotations across the unit tests codebase.

Here's what you could do:

1. Remove all annotations, not running any unit tests on CI, but have very extensive integration testing.
2. Only do opt-out (rather than opt-in) annotations for a small number of unit tests (e.g. at the class level rather than at the method level).

Reply:
Hi @fchollet,

Am I correct that instead I should use this mark for all test suites that are not passing now?

```
@pytest.mark.skipif(
    backend.backend() != "openvino",
    reason="OpenVINO backend does not support `some op`",
)
```

And should I remove `@pytest.mark.openvino_backend` in all tests?

Best regards,
Roman

Reply:
> Am I correct that instead I should use this mark for all test suites that are not passing now?

You should do this *if you enable the regular pytest CI for OpenVino*, which you should only do *if a small number of tests are failing*. If lots of tests are failing, you may be better off not enabling regular CI and instead rely on integration tests only.

Also, we might want to introduce single-line shortcuts to skip something with a given backend.

Reply:
responded below, please see